alf/algorithms/rl_algorithm.py

```diff
@@ -447,6 +447,23 @@ def summarize_train(self, experience, train_info, loss_info, params):
             if len(field) == 1:
                 summary_utils.summarize_distribution("action_dist", field[0])
 
+    @alf.configurable(whitelist=["custom_summary"])
+    def summarize_play(
+            self,
+            experience: Experience,
+            custom_summary: Optional[Callable[[Experience], None]] = None):
+        """Generate summaries for play or evaluate.
+
+        Args:
+            experience: experience collected from ``env.step`` of play.
```

Comment:
It is better to explain that the experience here is different from the one for summarize_rollout in that the experience here   is just for one step, where the one for summarize_rollout is a sequence.

Reply:
Added further docstring.